conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/data/ethereum/EthereumDataEndpoints.scala

```diff
@@ -34,12 +34,14 @@ trait EthereumDataEndpoints extends EthereumDataEndpointsCreator {
   def ethereumTokensEndpoint: Endpoint[((String, EthereumFilter), Option[String]), Option[List[QueryResponse]]] =
     tokensEndpoint(platform)
 
-  def ethereumTokenTransfersEndpoint: Endpoint[((String, EthereumFilter), Option[String]), Option[List[QueryResponse]]] =
+  def ethereumTokenTransfersEndpoint: Endpoint[((String, EthereumFilter), Option[String]), Option[
```

Comment:
comment more related to future PRs, i strongly recommend you to not formatted files where you don't add or include new changes.